subgraph-api/src/subgraph.ts

```diff
@@ -0,0 +1,401 @@
+// Copied from webapp/utils/subgraph.ts and slightly modified
+
+import config from 'config'
+import fetch from 'fetch-plus-plus'
+import { hemi, hemiSepolia } from 'hemi-viem'
+import {
+  type EvmDepositOperation,
+  type ToBtcWithdrawOperation,
+  type ToEvmWithdrawOperation,
+} from '../types/tunnel.ts'
+import { type Address, type Chain, checksumAddress as toChecksum } from 'viem'
+import { mainnet, sepolia } from 'viem/chains'
+
+type Schema = {
+  query: string
+  variables?: Record<string, string | number>
+}
+
+type SuccessResponse<T> = { data: T }
+type ErrorResponse = { errors: { message: string }[] }
+type GraphResponse<T> = SuccessResponse<T> | ErrorResponse
+
+type ConfigType = typeof import('../config/default.json')
```

Comment:
nice, you're mastering the mystic art of typing here 😎

Reply:
😎